carp.common/src/commonTest/kotlin/dk/cachet/carp/common/TimeSpanTest.kt

```diff
@@ -16,6 +16,20 @@ class TimeSpanTest
         assertEquals( 10.0, ten.totalMilliseconds )
     }
 
+    @Test
+    fun fromSeconds_with_whole_numbers_succeeds()
```

Comment:
You'll want property based testing here. Also a good way to find the max values

Reply:
Yeah, I need to find a better testing framework. :( That'd be a major rewrite, though.